README.rst

```diff
@@ -4,7 +4,7 @@
     :target: https://travis-ci.org/scikit-learn-contrib/lightning
 
 .. image:: https://ci.appveyor.com/api/projects/status/mmm0llccmvn5iooq?svg=true
```

Comment:
I guess this link is incorrect as well, but I don't know the right one to fix it.

Actual badge link can be found in the project settings at Appveyor site:

> You can see a badge image URL and sample markup on the **Badges** tab of project settings. 
    https://www.appveyor.com/docs/status-badges/

Reply:
Not entirely sure about this. @fabianp or @vene may have access to the appveyor admin.

Reply:
Hmm... Now I think that **status** link is correct. Looking at the README right now:

![image](https://user-images.githubusercontent.com/25141164/117514179-ef07f580-af9b-11eb-9721-dad237fb5c19.png)


I don't think that `pending` status can be shown for non-existing project... Let's wait several minutes and if it is correct, it should be green after the merged #167.

Reply:
> ... for non-existing project

If that project was `private` (`... or access denied` part in the message on the site) then without a token in link it cannot be possible to see different statuses right now.

Reply:
Now I'm sure this link is correct:

![image](https://user-images.githubusercontent.com/25141164/117515367-b9184080-af9e-11eb-9c75-e0a1afa68a36.png)